tasmota/xdrv_39_thermostat.ino

```diff
@@ -2022,8 +2022,16 @@ const char HTTP_THERMOSTAT_HL[]          PROGMEM = "{s}<hr>{m}<hr>{e}";
 
 #endif  // USE_WEBSERVER
 
-void ThermostatShow(uint8_t ctr_output)
+void ThermostatShow(uint8_t ctr_output, bool json)
 {
+  if (json) {
+    ResponseAppend_P(PSTR(",\"%s%i\":{"), "Thermostat", ctr_output); // this should probably stay static accross locales
+    ResponseAppend_P(PSTR("%s\"%s\":%i"), "", D_CMND_THERMOSTATMODESET, Thermostat[ctr_output].status.thermostat_mode);
```

Comment:
It. MUST be in english to satisfy JSON data handlers and was decided three years ago. Tasmota JSON is not localized.

Reply:
And I agree. That's what I meant by static, and the code currently only uses fixed identifiers in English only.

😂

Reply:
Let me know if I missed something, but it is already in English and will not get localized.

Reply:
You can just put the `"Thermostat"` part into the format string.

Reply:
@barbudor - 🤷‍♂️ Sure. Done.